Jenkinsfile_k8s

```diff
@@ -156,22 +195,22 @@ spec:
       environment {
         NETLIFY_AUTH_TOKEN = credentials('netlify-auth-token')
       }
+      post {
+        success {
+          recordDeployment("jenkins-infra", "plugin-site", "success", "https://deploy-preview-${CHANGE_ID}--jenkins-plugin-site-pr.netlify.app")
+        }
+        failure {
+          recordDeployment("jenkins-infra", "plugin-site", "failure", "https://deploy-preview-${CHANGE_ID}--jenkins-plugin-site-pr.netlify.app")
+        }
+      }
       steps {
         container('node') {
           script {
             def netlifySite = readJSON(text: sh(script: 'yarn exec netlify sites:list --json', returnStdout: true).trim()).find { it.name == "jenkins-plugin-site-pr" }
             env.NETLIFY_SITE_ID = netlifySite['site_id']
             env.NETLIFY_DOMAIN = netlifySite['default_domain']
-          }
-          sh('yarn exec netlify deploy --message "Preview deploy for ${CHANGE_ID}" --alias "deploy-preview-${CHANGE_ID}" -d ./plugins/plugin-site/public')
-          script {
-            def body = """<!-- DEPLOY PREVIEW -->✔️ Deploy Preview for jenkins-plugin-site-pr ready!\n\n😎 Browse the preview: https://deploy-preview-${CHANGE_ID}--jenkins-plugin-site-pr.netlify.app""".stripIndent()
-            def comment = pullRequest.comments.toList().find { it.body.contains("<!-- DEPLOY PREVIEW -->") }
-            if (comment != null) {
-              pullRequest.editComment(comment.id, body)
-            } else {
-              pullRequest.comment(body)
-            }
+
+            sh("""yarn exec netlify deploy --message "Preview deploy for ${CHANGE_ID}" --alias "deploy-preview-${CHANGE_ID}" -d ./plugins/plugin-site/public""")
```

Comment:
not multiline?

```suggestion
            sh("yarn exec netlify deploy --message "Preview deploy for ${CHANGE_ID}" --alias "deploy-preview-${CHANGE_ID}" -d ./plugins/plugin-site/public")
```

Reply:
You got double quotes inside double quotes. Now thinking about it. I could use single quotes on the outside and bash will handle it.